source/Halibut.Tests/Util/DataStreamExtensionMethods.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Halibut.Tests.Util
+{
+    public static class DataStreamExtensionMethods
+    {
+        // TODO: This could be async
+        public static string ReadAsString(this DataStream stream)
```

Comment:
This can be done in a future PR, or maybe @sburmanoctopus will just do it :D

Reply:
Later 😁 It's in a test assembly, so there's less value.